pages/notes.js

```diff
@@ -41,7 +42,7 @@ const Notes = () => {
   )
 
   return (
-    <>
+    <div>
```

Comment:
Why not use fragments? The idea of them is to avoid "wrapper hell", like this: 
![image](https://user-images.githubusercontent.com/36519478/83992587-6b223500-a916-11ea-8359-30853adf7497.png)

Also we can consider this topic for a Style Guide.